classes/sdimg.bbclass

```diff
@@ -39,21 +42,24 @@ IMAGE_BOOT_ENV_FILE ?= "uboot.env"
 # u-boot environment file
 IMAGE_UENV_TXT_FILE ?= "uEnv.txt"
 
+# These are usually defined in the machine section, but for daisy this
+# concept doesn't exist yet.
+IMAGE_BOOT_FILES ?= "u-boot.${UBOOT_SUFFIX}"
+IMAGE_BOOT_FILES_append_beaglebone ?= " MLO"
```

Comment:
IIRC `SOMEVAR ?= "foo"` makes sense for assignments only.

Make sure to keep the append as `SOMEVAR_append_beaglebone` (`beaglebone` coming in after `append`) though.